java8-build.gradle

```diff
@@ -9,7 +9,7 @@ java {
 }
 
 group = 'me.playbosswar.com'
-version = '8.7.5'
+version = '8.7.6'
```

Comment:
Each gradle build file also has the version in the publishing part of the file. Could you modify those as well?